sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoMessageSchema.java

```diff
@@ -1,125 +0,0 @@
-/*
```

Comment:
Do NOT delete!

Reply:
Why? This one is currently not used by anyone, or is this coming in a subsequent PR?
Notice that I have been refining access and removed dead code on the assumption that we should not be exposing code that we are not using (as part of the API/Schema refinements).